model.py

```diff
@@ -131,9 +133,10 @@ def build_model(self):
         self.complete_loss = self.contextual_loss + self.lam*self.perceptual_loss
         self.grad_complete_loss = tf.gradients(self.complete_loss, self.z)
 
+
     def train(self, config):
-        data = glob(os.path.join(config.dataset, "*.png"))
-        #np.random.shuffle(data)
+        data = glob(config.dataset)
```

Comment:
OK, I've put it back to just accepting a folder and inserting the glob in the code, but now it supports png and jpg/jpeg. Currently that's hardcoded, would you prefer the supported file types to be a parameter to the constructor instead?